quickstarts/maven/hello-world/README.md

```diff
@@ -0,0 +1,86 @@
+# Jkube Hello World Sample
```

Comment:
`s/Jkube/JKube/`

Please, describe the purpose of this example, e.g. "This is a minimal getting started project with the basics to start using Eclipse JKube..."